util/start_test

```diff
@@ -992,6 +992,19 @@ def set_up_executables():
 
     os.environ["NUMLOCALES"] = str(args.num_locales)
 
+    if args.max_locales == "0":
+        logger.write('[max_locales: "(default)"]')
+    else:
+        logger.write('[max_locales: "{0}"]'.format(args.max_locales))
+        os.environ["CHPL_TEST_MAX_LOCALES"] = str(args.max_locales)
```

Comment:
It's being used above for NUMLOCALES and other string and non-string args.  Are those incorrect too?